python/graphstorm/eval/eval_func.py

```diff
@@ -195,6 +210,38 @@ def labels_to_one_hot(labels, total_labels):
         one_hot[i,label]=1
     return one_hot
 
+def compute_hit_at_classification(preds, labels, k=100):
+    """ Compute hit@k for classification tasks
+
+        Parameters
+        ----------
+        preds : tensor
+            A 1-D tensor for single-label classification.
+        labels : tensor
+            A 1-D tensor for single-label classification.
+        k: int
+            Hit@K
+    """
+    assert len(preds.shape) == 2 \
+        and preds.shape[1] == 2 \
+        and len(labels.shape) == 1, \
+        "Computing hit@K for classification only works for binary classification tasks." \
+        "The preds must be a 2D tensor with the second dimension of 2. " \
+        "The labels must be a 1D tensor or a 2D tensor with the second dimension of 1"
+
+    # preds is a 2D tensor storing
+    # [probability of label 0, probability of label 1]
+    # 0 means negative, 1 means positive.
+    # We compute hit@K for positive labels
+    preds = preds[:,1]
+    if len(labels.shape) == 2:
```

Comment:
This line will not happen unless remove the `len(labels.shape) == 1` in line #227.